department-of-veterans-affairs/va.gov-cms
# VACMS-4446: Update vc cap address and hours fields.

## Description
See #4446

## Testing done
Visual / behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/109577675-00c4ac80-7ac4-11eb-8385-a0a4571a9d2b.png)

## QA steps
- As an administrator, go to `node/add/vet_center_cap` and visually verify that the changes reflected in the screenshot appear on the page.

As user _uid_ with _user_role_
1. Do this
1. Then that
1. Then validate Acceptance Criteria from issue
- [ ] This
- [ ] That
- [ ] The other thing

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
